src/components/Layout/Scripts.js

```diff
@@ -1,14 +1,35 @@
 import React from 'react'
 import Helmet from 'react-helmet'
 
-export default function Scripts () {
+export default function Scripts() {
   return (
     <Helmet>
       {/* HubSpot */}
-      <script id='hs-script-loader' async defer src='//js.hs-scripts.com/554916.js'></script>
+      <script
+        id="hs-script-loader"
+        async
+        defer
+        src="//js.hs-scripts.com/554916.js"
+      />
+      <script
+        charset="utf-8"
+        type="text/javascript"
+        src="//js.hsforms.net/forms/v2.js"
+      />
+
+      {/* HubSpot Form */}
+      <script type="text/javascript">{`
+        setTimeout(function () {
+        	hbspt.forms.create({
+            target: '#contentful-hubspot-form',
+          	portalId: "554916",
+          	formId: "0eb11a6a-7314-49b1-804b-56ee89a0e121"
+    	  	});
```

Comment:
Because the Hubspot script requires the `portalId` and `formId` here, it doesn't seem we can save these things into contentful like you have on the ConvertFlow form.

My understanding is:
Convertflow took an `id` saved in Contentful and passed it through to the div. Contentful's script then found this div and matched it to a form ID and inserted that form there.

However with Hubspot you're giving the ID to the script and the script is just finding whatever div you tell it to to anchor the form to.